site/js/template-tools.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * @Author: miaomiao612 dddoctorr612@gmail.com
+ * @Date: 2022-12-15 04:26:59
+ * @LastEditors: miaomiao612 dddoctorr612@gmail.com
+ * @LastEditTime: 2022-12-15 04:28:45
+ * @FilePath: \school-explorer\site\js\template-tools.js
+ * @Description: 这是默认设置,请设置`customMade`, 打开koroFileHeader查看配置 进行设置: https://github.com/OBKoro1/koro1FileHeader/wiki/%E9%85%8D%E7%BD%AE
+ */
```

Comment:
It looks like you have conflicts that are preventing the tests from running. I think if you remove this block of comments here it should resolve the conflicts.

```suggestion
```